server/jetstream_test.go

```diff
@@ -8895,6 +8895,38 @@ func TestJetStreamPubWithAsyncResponsePerf(t *testing.T) {
 	fmt.Printf("%.0f msgs/sec\n", float64(toSend)/tt.Seconds())
 }
 
+func TestJetStreamPubWithSyncPerf(t *testing.T) {
+	// Comment out to run, holding place for now.
+	//t.SkipNow()
```

Comment:
You need to put that back: it will probably fail all -race runs and even without race (the coverall run) takes too long:
```
=== RUN   TestJetStreamPubWithSyncPerf
time is 42.463880496s
23549 msgs/sec
--- PASS: TestJetStreamPubWithSyncPerf (42.49s)
```

Reply:
Thanks will do.